### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by aarmey**
- Make it easier to run Analyze with/without parallelization.
- Remove constant shape methods.
- Lower the upper bound on number of states.

I haven't been able to get this crash. @Farnazmdi, I think you mentioned it happened at the highest number of states, so I'm suggesting we just lower the number of states we evaluate in the AIC figure.

### CODE REVIEW

File: lineage/figures/figure12.py [hunk 1]
```diff
@@ -20,15 +20,6 @@
 tHMM_solver = tHMM(X=data[0], num_states=1)
 tHMM_solver.fit()
 
-constant_shape = [int(tHMM_solver.estimate.E[0].params[2]), int(tHMM_solver.estimate.E[0].params[4])]
```
Comment (aarmey) [inline lineage/figures/figure12.py:23]: This file isn't yet updated to run `atonce`.

Comment (Farnazmdi) [reply lineage/figures/figure12.py:23]: Thank you, I will fix this in my branch.

**Comment by Farnazmdi** [comment]
Thank you very much! 
I think the problem was with high concentration data, I am not sure if the high number of states makes problems or not, but my PR keeps failing because of that and I haven't merged it.

**Comment by aarmey** [comment]
This is set to merge into your branch.

Either way, evaluating fewer states will significantly speed up the figure build, which will help debugging.

**Comment by aarmey** [comment]
Actually, here figure 12 failed, which isn't even using the `atonce` fitting at the moment...

**Comment by aarmey** [comment]
https://github.com/meyer-lab/tHMM/pull/718/checks?check_run_id=1577850922

**Comment by Farnazmdi** [comment]
Thank you, I will look into figure 12.

**Pull request merged**